### DISCUSSION TITLE
Future plan about pricing

### DISCUSSION THREAD

**Discussion by jcubic** [sqlitecloud/discussions#7] [General]
I'm searching for a solution for hosting an SQLite database in the cloud for my Open Source project, [jQuery Terminal](https://terminal.jcubic.pl/#comments) where on a website I have a comments section. It's a single SQL table, that I need to migrate somewhere so I can use it with an API (Maybe I will add [another table for the demo](https://terminal.jcubic.pl/multiple-interpreters-demo.html) to migrate from MySQL). I plan to create a website that is self-contained and doesn't require any payment, so it will work when I die. This is important because I'm the only maintainer of the project, and I don't think I will be able to find anyone who will take over.

So the question is what are the future plans for pricing? Will you have a free tier for smaller projects with not a lot of traffic? My project has got almost 2k views in 2 weeks (it will be for database reads) but not a lot of writes, maybe 2-3 comments (writes) per month.

Do you think this small usage of the Open Source project would get a free tier? Or all tiers will be paid? This project is Open Source so you may not want to also give free tiers.

**Comment by btholt** [comment]
Hoping to have the pricing page out this week!

Long story short: there is a small free tier that will be made available, and a very reasonable $9.99/month tier for the next step up.

**Reply by casem** [reply]
I see that your pricing page has been published, and I noticed an issue with the information. On the pricing cards, the developer tier shows 3 GB of data sync, while the pricing table shows 10 GB.

**Reply by marcobambini** [reply]
Thanks a lot, @casem. We already fixed the issue. The Developer tier is now 10 GB.
